### ISSUE TITLE
Can CLI flags be passed to the programmatic API?

### ISSUE THREAD

**Issue by thasmo** [dotenvx/dotenvx#304]
For example, `--quiet` can not be set via programmatic API, afaik:

```ts
import dotenvx from '@dotenvx/dotenvx';

dotenvx.config({
    quiet: true, // not possible
});
```

Should other CLI flags be configurable via programmatic API as well?
Or is this possible and I am missing something?

Thank you!

**Comment by antoniosZ** [comment]
That definitely worked prior to v1.x and I think it may still work. It's just that the TS interface DotenvConfigOptions does not account for this and other CLI flags...

**Comment by thasmo** [comment]
So this may just be missing in the types?

**Comment by antoniosZ** [comment]
yes, that's what I think

**Comment by motdotla** [comment]
@getlarge maybe if you have time, do you want to take a crack at this? otherwise, anyone here could you create a PR to improve the type documentation.

**Comment by getlarge** [comment]
> @getlarge maybe if you have time, do you want to take a crack at this? otherwise, anyone here could you create a PR to improve the type documentation.

I can take care of it pretty soon.

**Comment by motdotla** [comment]
thanks @getlarge! released as 1.6.2

**Comment by thasmo** [comment]
Very nice! 👍
